refractor(patient invoice)set focus on quantity after selecting the code

jeremielodi opened the pull request:
closes #2837

jeremielodi committed:
refractor(patient invoice)set focus on quantity after the code selection

mbayopanda commented on client/src/modules/invoices/patientInvoiceForm.service.js:
```diff
@@ -433,7 +433,10 @@ function PatientInvoiceFormService(
+    // set focus on quantity field
+    setTimeout(() => {
+      document.getElementById(item.uuid).focus();
+    }, 100);
```
Could you use the angular version `$timeout`.

mbayopanda reviewed (commented):
LGTM

There is a comment on an Angularjs wrapper for setTimeout, which is more natural for an Angular application.

jeremielodi committed:
using angular timeout

jniles reviewed (commented):
Code looks okay to me.  The change from `setTimeout` -> `$timeout` was a good one.  You might consider doing the same thing with `document` -> `$document`.

I'm curious how you plan to write a test for this?

jniles committed:
test(Patient Invoice Form): unit test focus() call

This commit unit tests the focus() call in the Patient Invoice Form's
configureItem() call.

mbayopanda reviewed (approved):
LGTM 👍 

bors r+
